### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ms475**
I added new instructions that use micromamba instead of miniconda to set up the environment. Have verified that it is working on an instance of the cluster by doing a training run!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,27 +24,28 @@ These goals will be accomplished within the following pipeline structure:
...
     ```
     2) Create environment:
     ```
-    conda create -y --name cmap python=3.10
-    conda activate cmap
-    conda install pytorch torchvision pytorch-cuda=12.1 -c pytorch -c nvidia
-    pip install -r /home/YOUR_USERNAME/2024-winter-cmap/requirements.txt
+    micromamba create -y --name cmap python=3.10
+    micromamba activate cmap
+    micromamba install -y pytorch torchvision pytorch-cuda=12.1 -c pytorch-nightly -c nvidia
```
Comment (annawoodard) [inline README.md:44]: Even if pytorch nightly works in a simple test, let's avoid it since it will be less stable unless there is a specific need?

**Review by annawoodard — CHANGES_REQUESTED** [review]
This is fantastic but let's confirm we actually need nightly before merging—thanks!

**Review by cassietang717 — APPROVED** [review]

**Review by cassietang717 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ms475) [reply README.md:44]: I just retested with the stable version and it works! I will update the README with the modified code

**Review by annawoodard — APPROVED** [review]
Fantastic— thank you, @ms475!

**Pull request merged**